### DISCUSSION TITLE
Login in Darkmode

### DISCUSSION THREAD

**Discussion by root9191** [Leantime/leantime#1055] [General]
Hello, is there a way to put the Login Screen to Darkmode too?

**Comment by gloriafolaron** [comment]
Hi! This is not currently enabled.

Where you expect to make this change or would you expect this to change along with the overall setting?

**Reply by root9191** [reply]
i would change it with the overall Setting :)

**Reply by gloriafolaron** [reply]
That'd be my own personal bias, too, but I'm a dark mode fiend. (Marcel lives in the light). I'll follow up with him and see what sneaking this in on another upcoming patch would take. Thanks for pointing it out!

**Reply by root9191** [reply]
Ok thank you :)

**Reply by gloriafolaron** [reply]
So one thing that came up was around the recall of what the set user's preference was for the system mode. One of the primary ways to manage this is through cookies but that isn't always favorable. 

Other routes have a significant amount of complexity around them but haven't forgotten. If you think of or know of any approaches you've used or experienced in the past, we'd be happy to discuss.

Thanks again!

**Reply by root9191** [reply]
Sorry i don't know if i understood this right 😅
So you are looking for a Function to declare the Login Page with Dark Mode? On LLC (LittleLinkCustom, selfhosted Linktree) for example i type "#dark" at the end of the URL and that get's my Loginscreen also to Dark Mode.

**Reply by gloriafolaron** [reply]
Thanks! That was helpful.

The new release went out and had a fix for the dark mode login screen.

https://github.com/Leantime/leantime/discussions/1065

**Reply by root9191** [reply]
Thank you looks good so far :)
Will there be a possibility to overlay 2 or more Projects with the Kanboard View?

**Reply by gloriafolaron** [reply]
@marcelfolaron you might be better for the kanban question here 😄

**Comment by marcelfolaron** [comment]
Can you update to the latest version. You might have to go to your profile and just hit save to make sure the theme cookie gets created

**Comment by marcelfolaron** [comment]
There are currently no plans to do a global Kanban view. In my opinion the
Kanban view is a “work-in-progress” view for a project and individual team.
It should be fairly small in terms of numbers of tickets.

I’d love to learn more about your use case? Would it be feasible for you to
move projects as milestones within one project?
